js/app.js

```diff
@@ -1,129 +1,172 @@
 'use strict';
 console.log('app.js file is connected');
 
-let points = 0
+let points = 0;
 
 let username = prompt("Hi! You've reached Mike's page. What's your name?"); 
     alert('Welcome ' + username + '!');
 
 alert('Please answer yes or no.');
 
-let answer1 = prompt('Was I born in 1999?').toLowerCase();
-console.log('user input', answer1);
+function questionOne(){
+    let answer1 = prompt('Was I born in 1999?').toLowerCase();
+    console.log('1. user input', answer1);
 
-if (answer1 === "yes") {
-    alert('Nope.');
-}
-if (answer1 === "no") {
-    alert('Whoop there it is!');
-    points += 1;
+    if (answer1 === "yes") {
+        alert('Nope.');
+        console.log(username + 'still has ' + points + ' points');
+    }
+    if (answer1 === "no") {
+        alert('Whoop there it is!');
+        points += 1;
+        console.log(username + ' now has ' + points + ' points');
+    }
 }
 
-let answer2 = prompt('Did I work as a valet?').toLocaleLowerCase();
-console.log('user input', answer2);
+questionOne();
 
-if (answer2 === "yes") {
-    alert('Whoop yes I did!');
-    points += 1;
+function questionTwo(){
+    let answer2 = prompt('Did I work as a valet?').toLocaleLowerCase();
+    console.log('2. user input', answer2);
+
+    if (answer2 === "yes") {
+        alert('Whoop yes I did!');
+        points += 1;
+        console.log(username + ' now has ' + points + ' points');
 }
-if (answer2 === "no") {
-    alert("It's a no from me dawg.");
+    if (answer2 === "no") {
```

Comment:
I recommend making this an else if and moving it up to be on the same line as the closing curly bracket on 36.